src/App.js

```diff
@@ -18,11 +18,11 @@ const router = createBrowserRouter([{
   element: <AppLayout />,
   children: [
     {
-      path: "/",
+      path: "/main",
```

Comment:
The default url "/" should point to Main component. If Auth is not present in the Main component, it should redirect to "/login" route.